Lib/concurrent/futures/process.py

```diff
@@ -718,12 +733,12 @@ def shutdown(self, wait=True, *, cancel_futures=False):
         with self._shutdown_lock:
             self._cancel_pending_futures = cancel_futures
             self._shutdown_thread = True
+            if self._executor_manager_thread_wakeup is not None:
```

Comment:
I replaced ``if self._executor_manager_thread:`` with ``if self._executor_manager_thread_wakeup is not None:``. Or was the previous code written on purpose?

cc  @pitrou @tomMoral

Reply:
I think this is the same as both are set to `None` in the same call to shutdown. (L.744/750)
This is more explicit this way so perfect.